app/views/curation_concerns/base/_relationship.html.erb

```diff
@@ -0,0 +1,36 @@
+<h3><% if presenters.first.citi_presenter? %>
```

Comment:
I agree. This is still manageable so I left it in one file for now. If we want more customization we can split it up.